packages/palette/src/elements/Carousel/Carousel.tsx

```diff
@@ -52,7 +52,9 @@ CarouselRail.defaultProps = {
 export type CarouselCellProps = BoxProps
 
 /** A `CarouselCell` wraps a single child in the carousel */
-export const CarouselCell = styled(Box)``
+export const CarouselCell = styled(Box)`
+  white-space: normal;
```

Comment:
🤔 how was this not the default previously? Box doesn't set this explicitly right?

Reply:
The outer rail sets it to `nowrap`